source/DistanceCalculationSettings.cpp

```diff
@@ -57,6 +58,21 @@ void DistanceCalculationSettings::Load(const DataNode &node)
 
 
 
+void DistanceCalculationSettings::Save(DataWriter &out) const
+{
+	if(wormholeStrategy == WormholeStrategy::NONE)
+		out.Write("no wormholes");
+	else if(wormholeStrategy == WormholeStrategy::ONLY_UNRESTRICTED)
+		out.Write("only unrestricted wormholes");
+	else if(wormholeStrategy == WormholeStrategy::ALL)
+		out.Write("all wormholes");
+
+	if(assumesJumpDrive)
+		out.Write("assumes jump drive");
+}
```

Comment:
How is this related to the new feature? I don't see any old code for wormholing strategy being removed in this PR.

Reply:
`LocationFilter::Save` calls `LocationFilter::NearFilter::Save` which needs to be able to save distance calculations options. That code can either go here, or in that method. I put it here.